### PR TITLE
Ruby style fixes

### PR DESCRIPTION
**Pull request by jmuk**
Those styles are from gcloud-ruby:
- prefers "" rather than ''
- do not use () for def
- use : instead of => for simbol-keyed Hashes
- add rubocop suppression for line length and method length:

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline [hunk 1]
```diff
@@ -50,133 +54,133 @@ module Library
...
+          "next_page_token",
+          "strings")
       }.freeze
 
       private_constant :PAGE_DESCRIPTORS
 
       BUNDLE_DESCRIPTORS = {
-        'publish_series' => Google::Gax::BundleDescriptor.new(
-          'books',
+        "publish_series" => Google::Gax::BundleDescriptor.new(
```
Comment (geigerj) [inline src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline]: `=>` here (and for `PAGE_DESCRIPTORS`)

Comment (jmuk) [inline src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline]: Actually this is correct. This is confusing, but in Ruby, `{"foo": "bar"}` and `{"foo" => "bar"}` are different. 
- `{"foo" => "bar"}`: normal one, the key is as is seen (in this example, a String instance `"foo"`)
- `{"foo": "bar"}`: special syntax, the key is converted into a Symbol (in this example, `:foo`)

For page descriptors and bundle descriptors, we want to have string-keyed Hash, so this still needs to be `=>`.

Comment (geigerj) [inline src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline]: Ah, I see. Thanks.

**Comment by geigerj** [comment]
LGTM

**Pull request merged**